runelite-api/src/main/java/net/runelite/api/events/player/headicon/OverheadPrayerChanged.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.api.events.player.headicon;
+
+import lombok.Getter;
+import net.runelite.api.HeadIcon;
+import net.runelite.api.Player;
+
```

Comment:
remove constructor, add `@Data`, dont extend PlayerHeadIconChanged, just add the actor in